### PR TITLE
Preprod: movie: Immortal (2004) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Wacky alien/god trope overload. Rating 2/5.

Notes:
- The use of **what if I had multiple personalities** is a bit iffy.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -4446,6 +4446,55 @@ fear of commitment [Clementine accused Joel of this shortcoming.]
...
+2004-03-24
+
+:: References
+https://en.wikipedia.org/wiki/Immortal_(2004_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+coping with mortality [The god/alien Horus was determined to mate with a human female after being stripped of his immortality. Having offspring was presumably his way of coping with the reality that he would eventually die.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:4476]: What about:

https://www.themeontology.org/theme.php?name=the%20quest%20for%20immortality

https://www.themeontology.org/theme.php?name=immortality

https://www.themeontology.org/theme.php?name=coping%20with%20mortality

https://www.themeontology.org/theme.php?name=the%20desire%20to%20have%20children

Sounds like they are in the ballpark?

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 2]
```diff
@@ -4446,6 +4446,55 @@ fear of commitment [Clementine accused Joel of this shortcoming.]
...
+speculative genetic engineering [The story is set in a a future where genetically altered humans lived side by side with unaltered men and women. Much was made of the heroine Jill being some sort of genetically engineered person.]
+Ancient Egyptian mythology [The story focused on the Ancient Egyptian god Horus, and how he lost his immortality.]
+extraterrestrial being [A revelation of the film is that the ancient Egyptian gods were really aliens.]
+the god is an alien [A revelation of the film is that the ancient Egyptian gods were really advanced technology wielding aliens.]
+aliens impregnating human females [The god Horus, who was actually an alien, made a number of clandestine attempts to impregnate Jill.]
+romantic love [Jill and Nikopol ended up falling in love.]
+what if I had multiple personalities [The film explores Nikopol having two selves: His ordinary self and the rapist self he became whenever Horus inhabited his body.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:4483]: 🤔

**Review by odinlake — APPROVED** [review]
Suggestions inline

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:4476]: On the immortality front, I deemed the story most similar to [Deja Q](https://themeontology.org/story.php?name=tng3x13) in which we elected to use only **coping with mortality**. I suppose it'd make sense to add the newly coined **immortality** theme to both stories though.

Good catch with **the desire to have children**. It has been added.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:4483]: I think the problem is we don't have a theme to the effect of "what if my body became inhabited by another entity". We do, however, have these the related themes **spirit possession**, demonic possession**, **what if a loved one became possessed by another being**, and **evil hand**.

**Review by odinlake — APPROVED** [review]

**Pull request merged**